Sources/Rendering/Core/CellPicker/index.js

```diff
@@ -244,6 +254,88 @@ function vtkCellPicker(publicAPI, model) {
     return tMin;
   };
 
+  publicAPI.intersectVolumeWithLine = (p1, p2, t1, t2, tol, actor, mapper) => {
+    let tMin = Number.MAX_VALUE;
+
+    const imageData = mapper.getInputData();
+    const origin = imageData.getOrigin();
+    const spacing = imageData.getSpacing();
+    const dims = imageData.getDimensions();
+    const scalars = imageData.getPointData().getScalars().getData();
+    const extent = imageData.getExtent();
```

Comment:
this should also consider the imageData `direction`.

Reply:
How do I add the information in the code?

Reply:
there must be some kind of a matrix inversion to apply to the line/ray I believe.

Reply:
I think p1 and p2 defines the ray and direction. That´s why always t, the parametric parameter is always positive between 0 and 1

Reply:
Please create an example with headsq.vti, it has a non identity orientation.

Reply:
Add the example. Yes you are right!!!!